nncf/onnx/graph/node_utils.py

```diff
@@ -53,3 +56,21 @@ def get_bias_value(node_with_bias : NNCFNode, model: onnx.ModelProto) -> np.ndar
     if metatype == ONNXIdentityMetatype:
         return onnx_graph.get_initializers_value(node.input[0])
     raise RuntimeError('Could not find the bias value of the node')
+
+
+def get_nncf_input_node_next_onnx_nodes(nncf_graph: NNCFGraph) -> Dict[str, Tuple[str, int]]:
```

Comment:
The method's name is weird. It would be better to call it `get_input_edges_mapping` or something like that.

Reply:
Done